src/hornedBeast.js

```diff
@@ -0,0 +1,56 @@
+import { Component } from 'react';
+import  Container from 'react-bootstrap/Container';
+import beastBios from './data.json';
+import  Button from 'react-bootstrap/Button';
+import  Modal from 'react-bootstrap/Modal';
+import Image from 'react-bootstrap/Image';
+
+
+
+class hornedBeastM extends Component {
```

Comment:
Nice work so far, you idea to use map() in Main is for sure a good idea. Here, I would consider renaming this to `hornedBeastModal` as at a glance, m is a bit too vague.